README.md

```diff
@@ -1,8 +1,17 @@
-# Beard Template Engine
+# Beard: A Blazing-Fast Template Engine
 
 [![Build Status](https://travis-ci.org/zalando/beard.svg)](https://travis-ci.org/zalando/beard)
 
-Blazing fast, [open source](https://github.com/zalando/beard), logic-less template engine written in Scala, used to parse templates using a simple syntax:
+Beard is an open-source, logic-less templating engine, written in Scala and inspired by [Mustache](https://mustache.github.io/). You can use it out-of-the-box; it has no dependencies (see Requirements list below). 
```

Comment:
We shouldn't specify that it doesn't have dependencies, as it has some. For example the streaming library.